src/components/AudioAnalysis/SpAnalysisPitchSelect.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Checkbox from '@material-ui/core/Checkbox';
+import { FormGroup, FormControlLabel, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    color: '#1ED760',
+    '&$checked': {
+      color: '#1ED760',
+    },
+  },
+}));
+
+export default function PitchSelect(props: any) {
+  const pitchClasses = [
```

Comment:
I saw you need this pitchClasses in another file. We could put them in a seperate file and import them where needed

Reply:
Fixed in latest commit